app/main.py

```diff
@@ -1 +1,56 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+        self, name: str,
+        appetite: int,
+        is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(
+        self, name: str,
+        is_hungry: bool = True,
+        appetite: int = 3
+    ) -> None:
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(
+        self, name: int,
+        is_hungry: bool = True,
+        appetite: int = 7
+    ) -> None:
```

Comment:
```suggestion
    def __init__(
        self, 
        name: int,
        is_hungry: bool = True,
        appetite: int = 7
    ) -> None:
```

Fix line wrapping, also in other similar cases

Reply:
got it thanks